server/models/Contest.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose')
+const { User } = require('./User')
+
+const contestSchema = new mongoose.Schema({
+    title: {
+        type: String,
+        required: true
+    },
+
+    description: {
+        type: String,
+        required: true
+    },
+
+    prizeAmount: {
+        type: Number
+    },
+
+    deadline: {
+        type: Date,
+    },
+
+    created: {
+        type: Date
+    },
+
+    user: {
```

Comment:
Please see link on how to reference another model in mongoose: 
https://alexanderzeitler.com/articles/mongoose-referencing-schema-in-properties-and-arrays/

Reply:
ok